infrastructure/kube/keep-test/monitoring/grafana-service.yaml

```
@@ -0,0 +1,17 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: grafana
+  namespace: monitoring
+  annotations:
+      prometheus.io/scrape: 'true'
+      prometheus.io/port: '3000'
+spec:
+  selector:
+    app: grafana
+  type: NodePort
+  ports:
+    - port: 3000
+      targetPort: 3000
```

We named the port in the deployment 
```yaml
          ports:
            - name: grafana
              containerPort: 3000
```
so maybe we could use the name here?
```suggestion
      targetPort: grafana
```